mflowgen/common/power-domains/outputs/dont-touch-constraints.tcl

```diff
@@ -5,8 +5,19 @@
 # Don't touch constraints 
 #----------------------------------------------------------------
 
+
+set_dont_touch [get_cells -hierarchical *u_mux_logic*] false
```

Comment:
It would be good to add a comment about the specific problem you are trying to avoid here (i.e., don't allow buffering before the AND gate, and allow sizing of the AND gate).